### PR TITLE
#2 Added links to GitHub profile/pages

### PR DESCRIPTION
**Pull request by JoeFoster-cn**
Updated the README files with URLs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 Nicholas Ferrari [Github](https://github.com/NickFerra), [GitHub Pages Website](https://nickferra.github.io/NickWeb/)
 
 Jennifer Carey - [Click here](https://github.com/jennifer-carey) to see my GitHub profile and [click here](https://jennifer-carey.github.io/) to see my GitHub Pages.
+[Click here](https://github.com/JoeFoster-cn) to see my GitHub profile and [click here](https://joefoster-cn.github.io/) to see my GitHub Pages.
```
Comment (mafromist) [inline README.md:5]: Maybe add your name to the beginning of the line

**Review by mafromist — DISMISSED** [review]
Maybe add your name to the beginning of the line, but looks okay

File: README.md (hunk 1 above)
Comment (JoeFoster-cn) [reply README.md]: I thought I had, good spot! I've just fixed it

**Review by JanisMc — APPROVED** [review]

**Review by LRWright — APPROVED** [review]
👍

**Review by Connell97 — APPROVED** [review]
Excellent use of your name!

**Pull request merged**